src/Apps/CollectorProfile/collectorProfileRoutes.tsx

```diff
@@ -198,7 +188,8 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
         `,
       },
       {
-        path: "saves2",
+        path: "saves2/:id?",
+        ignoreScrollBehavior: true,
```

Comment:
**question:** What does `ignoreScrollBehavior` do?

Reply:
this helps us to keep scroll offset when switching to other collections.

#### with `ignoreScrollBehavior`
https://user-images.githubusercontent.com/3513494/218459283-12631002-46bc-45be-b134-d491ecafac0d.mp4

#### without `ignoreScrollBehavior`
https://user-images.githubusercontent.com/3513494/218459446-f43df1b9-b790-42ad-9ae4-f8e4960fb786.mp4

Reply:
Ah, thank you for clear explanation + screencaps 🙏🏽

Reply:
you're welcome 👌